lib/android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -846,4 +853,64 @@ public void onPanDrag(MotionEvent ev) {
         WritableMap event = makeClickEventData(coords);
         manager.pushEvent(context, this, "onPanDrag", event);
     }
+    @Override
+    public void onIndoorBuildingFocused() {
+      IndoorBuilding building = this.map.getFocusedBuilding();
+      if (building != null) {
+        List<IndoorLevel> levels = building.getLevels();
+        int index = 0;
+        WritableArray levelsArray = Arguments.createArray();
+        for (IndoorLevel level : levels) {
+          WritableMap levelMap = Arguments.createMap();
+          levelMap.putInt("index", index);
+          levelMap.putString("name", level.getName());
+          levelMap.putString("shortName", level.getShortName());
+          levelsArray.pushMap(levelMap);
+    		}
```

Comment:
the code formatting is broken here. Please match the rest formatting from the rest of the file (4 spaces indentation)